src/Playwright/Core/BrowserContext.cs

```diff
@@ -44,6 +44,7 @@ internal class BrowserContext : ChannelOwner, IBrowserContext
     private readonly Dictionary<string, Delegate> _bindings = new();
     private readonly BrowserContextInitializer _initializer;
     private readonly Tracing _tracing;
+    internal readonly HashSet<IPage> _backgroundPages = new();
```

Comment:
Why does this use a HashSet intead of List?

Reply:
Hmm, was following [upstream impl](https://github.com/microsoft/playwright/blob/main/packages/playwright-core/src/client/browserContext.ts#L61), doesn't really matter probably, Java uses a [List](https://github.com/microsoft/playwright-java/commit/452effbe3f81c596d6f40c02b236efde290c90bf#diff-24e3a8cd1a7d4f1c30ddf3f3550359d3d9ad77c879e1b5f51a5b7b7f3c8d8ad9R50) as well.